java/kotlin/BUILD

```diff
@@ -0,0 +1,36 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library")
+
+KT_LITE_SRCS = [
+    "src/main/kotlin/com/google/protobuf/DslList.kt",
+    "src/main/kotlin/com/google/protobuf/DslMap.kt",
+    "src/main/kotlin/com/google/protobuf/DslProxy.kt",
+    "src/main/kotlin/com/google/protobuf/ExtendableMessageLiteExtensions.kt",
+    "src/main/kotlin/com/google/protobuf/ExtensionList.kt",
+    "src/main/kotlin/com/google/protobuf/OnlyForUseByGeneratedProtoCode.kt",
+    "src/main/kotlin/com/google/protobuf/ProtoDslMarker.kt",
+    "src/main/kotlin/com/google/protobuf/UnmodifiableCollections.kt",
+]
+
+kt_jvm_library(
+    name = "kt_lite",
```

Comment:
unless there's a stylistic reason to do this, I'd drop the kt_ prefix, since the target path contains "kotlin" in the name already.